project_electron/orgs/signals.py

```diff
@@ -0,0 +1,19 @@
+import pwd
+import os
+
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from orgs.models import Organization
+
+from transfer_app.RAC_CMD import delete_system_group
+
+@receiver(pre_delete, sender=Organization)
+def delete_organization(sender,instance,**kwargs):
+
+	# update group of upload path to root
+	root_uid = pwd.getpwnam('root').pw_uid
+	os.chown(instance.org_machine_upload_paths()[0],root_uid,root_uid)
```

Comment:
@dboutmybizness I'm getting an error on this line when running unit tests, I believe because the upload directory is removed [during testing](https://github.com/RockefellerArchiveCenter/aurora/blob/master/project_electron/transfer_app/test/test_transfer_routine.py#L46)? Would a simple if statement to check for the existence of the directory before attempting to chown be sufficient?